Foundation/NSAffineTransform.swift

```diff
@@ -9,7 +9,7 @@
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
```

Comment:
Probably true, but what does that accomplish for us besides making us put another import in our files?